noxfile.py

```diff
@@ -151,7 +159,7 @@ def run_tests(session):
     set_environment_variables(PYBAMM_ENV, session=session)
     session.install("setuptools", silent=False)
     session.install("-e", ".[all,dev,jax]", silent=False)
-    session.run("python", "run-tests.py", "--all")
+    session.run("python", "-m", "pytest", "-svra")
```

Comment:
```suggestion
    session.run("python", "-m", "pytest")
```

This command will run not only the unit and integration tests but also the example scripts, which contradicts the docstring for the function. Can we change it to `pytest --unit --integration` or similar?